status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.175.4",
-    "commit-sha1": "5304406079a5e3e1cfea285fef892ac56f2c3aad",
-    "src-sha256": "08xprnmpw0nz2zjm1wp8ypjhdyfvgmh7pqrlk69prgr23xa31ady"
+    "version": "5a2e3501e53541529fce849baca9dc5ca48a9e76",
```

Comment:
Quick tip: we can use the status-go branch `peer-syncing-toggle` directly in the script.